superdesk/io/commands/update_ingest.py

```diff
@@ -259,6 +259,26 @@ def process_anpa_category(item, provider):
         raise ProviderError.anpaError(ex, provider)
 
 
+def derive_category(item, provider):
+    """
+    Assuming that the item has at least one itpc subject use the vocabulary map to derive an anpa category
+    :param item:
+    :return: An item with a category if possible
+    """
+    try:
+        categories = []
+        map = superdesk.get_resource_service('vocabularies').find_one(req=None, _id='iptc_category_map')
+        if map:
+            for subject in item.get('subject', []):
+                for map_entry in map['items']:
+                    if map_entry['is_active'] is True and subject['qcode'] == map_entry['subject']:
```

Comment:
I think `get_resource_service('vocabularies').find_one(req=None, _id='iptc_category_map')` doesn't return `is_active`. The call gives all active vocabularies without is_active flag.